src/test/resources/features/launchInfo.feature

```diff
@@ -0,0 +1,19 @@
+@api
+Feature: Exact launch info checking
+
+  Background: User is Logged In
+    Given use is logged in
+
+  Scenario: Get dashboard name from the project
```

Comment:
This scenario is about Dashboards. Maybe better remove them to a separate feature file?

Reply:
done